sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/EventHubAsyncClientIntegrationTest.java

```diff
@@ -129,6 +129,12 @@ void getPropertiesWithCredentials() {
         // Arrange
 
         // Act & Assert
+        logger.info(this.getClass().getName()  + "getPropertiesWithCredentials " + "AZURE_EVENTHUBS_FULLY_QUALIFIED_DOMAIN_NAME : " + System.getenv("AZURE_EVENTHUBS_FULLY_QUALIFIED_DOMAIN_NAME"));
+        logger.info(this.getClass().getName()  + "getPropertiesWithCredentials " + "AZURE_EVENTHUBS_EVENT_HUB_NAME : " + System.getenv("AZURE_EVENTHUBS_EVENT_HUB_NAME"));
+        logger.info(this.getClass().getName()  + "getPropertiesWithCredentials " + "AZURE_CLIENT_ID : " + System.getenv("AZURE_CLIENT_ID"));
+        logger.info(this.getClass().getName()  + "getPropertiesWithCredentials " + "AZURE_CLIENT_SECRET : " + System.getenv("AZURE_CLIENT_SECRET"));
```

Comment:
We do not in any circumstances want to be logging secrets (here and a few places below).

Reply:
The above logging is only used to investigate the ci exception, and will be remove after the investigation.

Reply:
Meaning you intend to only run it locally? If it's ever run in CI, that secret is now exposed in the logs and needs to be rotated.

Reply:
Okey, The above Logging has been removed

Reply:
Thank you!

Reply:
That's welcome.